src/encoder.rs

```diff
@@ -2281,12 +2285,14 @@ fn encode_partition_bottomup(seq: &Sequence, fi: &FrameInvariants, fs: &mut Fram
     let mut subsize: BlockSize;
 
     // Code the whole block
+    // TODO(yushin): Try move PARTITION_NONE to below partition loop
     if !must_split {
```

Comment:
I wonder if this would be better moved inside the partition type loop.

Reply:
As I commented in https://github.com/xiph/rav1e/pull/816/files/b8ea05b8aa72cd66d4711dad2ab07b0d0d449401#r241243259,
I plan to think about that, but the reason why I keep the block of code for PARTITION_NONE case is that only that case actually calls rdo_mode_decision() because it is the terminal (i.e. not splitable literally) in terms of partitioning. Also, pmv (predicted ME centers) are used for only that case, and preparing the partitions are NOT used for that case.

Reply:
Oops, the comment was even in the view. Sorry I missed it.